packages/dotcom/src/requests.ts

```diff
@@ -2,7 +2,6 @@ import { BannerPayload, EpicPayload, HeaderPayload } from '../../shared/src/type
 import { TestTracking } from '../../shared/src/types/abTests/shared';
 
 export interface ModuleData {
-    url: string;
```

Comment:
DCR no longer uses the url field. This was previously for telling the DCR page where to fetch the component from. But the components now live in DCR